lib/utils.ts

```diff
@@ -40,6 +40,8 @@ export const api = () => apiInstance;
 
 export const isProd = () => process.env.NEXT_PUBLIC_VERCEL_ENV === "production";
 
+export const isE2E = () => process.env.E2E === "true";
```

Comment:
`NEXT_PUBLIC_` isn't needed here?

Reply:
As discussed saving for future PR if needed